elrax/elrax
# feat: add eas configuration and dev deployments

## What does this PR do?

- Adds EAS configuration for development deployments.
- Adds `expo-dev-client` package.
- Updates `pnpm-lock.yaml`.
- Fixes `tailwindcss` version.
- Alters `.env.example` file.

## Type of change


## How should this be tested?

- Should add env variables to the Expo, test it, then add to Github Secrets and test only with them.
